src/features/settings/appearance/PostView.tsx

```diff
@@ -12,7 +12,7 @@ export default function PostView() {
   return (
     <>
       <ListHeader>
-        <IonLabel>Posts</IonLabel>
+        <IonLabel>Posts {typeof postsAppearanceType}</IonLabel>
```

Comment:
Why is this changed?

Reply:
Good catch, left it in whilst debugging, Will remove and fix, the other things you mentioned. Thanks!